src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/Flows/AcquireTokenHandlerBase.cs

```diff
@@ -376,14 +383,21 @@ private void LogReturnedToken(AuthenticationResult result)
             {
                 string accessTokenHash = CryptographyHelper.CreateSha256Hash(result.AccessToken);
 
-                CallState.Logger.Information(this.CallState,
-                    string.Format(CultureInfo.CurrentCulture,
-                        "=== Token Acquisition finished successfully. An access token was retuned:\n\tAccess Token Hash: {0}\n\tExpiration Time: {1}\n\tUser Hash: {2}\n\t",
-                        accessTokenHash,
-                        result.ExpiresOn,
-                        result.UserInfo != null
-                            ? CryptographyHelper.CreateSha256Hash(result.UserInfo.UniqueId)
-                            : "null"));
+                if (LoggerCallbackHandler.PiiLoggingEnabled)
+                {
+                    CallState.Logger.InformationPii(this.CallState,
+                        string.Format(CultureInfo.CurrentCulture,
+                            "=== Token Acquisition finished successfully. An access token was returned: Access Token Hash: {0}\n\tExpiration Time: {1}\n\tUser id: {2}\n\t",
```

Comment:
update the log message to not log access token hash and uniqueID. these values are not very helpful anyways.